openhcl/virt_mshv_vtl/src/processor/tdx/mod.rs

```diff
@@ -876,6 +879,17 @@ impl UhProcessor<'_, TdxBacked> {
                     .lapic
                     .request_fixed_interrupts(irr);
             }
+        };
+
+        // TODO TDX: filter proxy IRRs.
+        if let Some(irr) = proxy_irr {
```

Comment:
As mentioned in the kernel review, it would be good to move proxy IRR handling to the kernel now that we have the machinery to do so.

Reply:
Done, user-space can block specific vectors by setting the bit to 1 in the `proxy_irr_blocked` field of the run page.